src/components/Form/form.module.css

```diff
@@ -62,32 +64,36 @@ limitations under the License.
  */
 
 .control {
-  --cpd-form-border-color: var(--cpd-color-gray-600);
-  --cpd-form-bg: var(--cpd-color-bg-canvas);
-
-  border: 1px solid var(--cpd-form-border-color);
-  background: var(--cpd-form-bg);
-  border-radius: 8px;
+  border: 1px solid var(--cpd-color-border-interactive-primary);
+  background: var(--cpd-color-bg-canvas);
+  border-radius: 0.5rem;
   padding: var(--cpd-space-3x) var(--cpd-space-4x);
+  box-sizing: border-box;
 }
 
-.control:hover {
-  --cpd-form-border-color: var(--cpd-color-gray-1100);
+/* Disabling the descending specificity rule to allow for selectors like below
+ * It is using the "fake" element hack to style checkboxes, radio, mfa codes ...
+ */
```

Comment:
I believe you can move the `stylelint-disable` line down here so it's next to this comment